packages/storybook/stories/css-component/combining-components/inline-link.stories.mdx

```diff
@@ -12,6 +12,7 @@ export const ParagraphWithLink = ({ textContent, href, link }) =>
 
 <Meta
   title="CSS Component/Combining components/Link"
+  component={ParagraphWithLink}
   argTypes={{
     href: {
       description: "URL",
```

Comment:
I'm getting the following error on this page:
`TypeError: textContent is undefined`